src/add_default_plugins.rs

```diff
@@ -14,17 +14,27 @@ impl AddDefaultPlugins for AppBuilder {
         self.add_plugin(bevy_window::WindowPlugin::default());
         self.add_plugin(bevy_asset::AssetPlugin::default());
         self.add_plugin(bevy_scene::ScenePlugin::default());
-        self.add_plugin(bevy_render::RenderPlugin::default());
+
+        #[cfg(feature = "bevy_render")]
+        self.add_plugin(bevy_gltf::GltfPlugin::default());
```

Comment:
This statement (and the one with the `bevy_gltf` feature below) seem swapped?

Reply:
Indeed!
Fixed.